docs/introduction.mdx

```diff
@@ -9,56 +9,120 @@ import SolanaIcon from '@site/src/assets/SolanaIcon'
 
-</Grid>
+### Solution 
+ 
+Liquid staking allows using the staked crypto assets in other trading or investing opportunities to let the user get the best of both worlds - a reward on your staked assets, as well as the returns from new trading/investing opportunities that you spot. Liquid staking introduce various fundamental benefits by: 
+- Allowing users to earn staking rewards without fully locking their `st[token]`;  
+- Making it possible to earn rewards on as small a deposit as users want (i.e, Ethereum requires minimum 32 `ETH` staked);  
+- Providing the `st[token]` a building block for other applications and protocols (e.g., as collateral in lending or other trading DeFi solutions);  
+- Providing an alternative to exchange staking, self-staking, and other semi-custodial and decentralised protocols. 
+ 
+### Advantages over self staking and exchange staking 
+ 
+**Self-staking** is great, but it comes with some disadvantages. Setting up a validator node requires a pristine technical understanding; brings with it a minimum deposit of 32 ETH; slashing and offline penalties can get very severe if the staking is managed improperly; the staked amount is locked up for a significant period. Eventually, it leads to a small number of stakers, which is bad for any PoS ecosystem. 
+ 
+Alternatively, it may be possible to produce staking through **centralised exchanges**. First of all, crypto assets and CeFi are not suited well together from the fundamental standpoint.. It is also worth mentioning the economic aspect - by staking within an centralised entity, the user does not receive a corresponding token in return and, thus, loses the opportunity to perform any subsequent activity within DeFi or the same centralised entity, where assets were staked. Yes, APR, when staking on centralised exchanges might be higher, but with a significant amount aggregated within the centralised entity comes a huge potential influence to the ecosystem that was fundamentally designed decentralised.  
+ 
+Through the use of a **liquid staking** solution such as Lido, users can eliminate these inconveniences and benefit from secure, non-custodial staking backed by industry leaders.
+
+## Security
+
+Lido is a secure liquid staking solution for a number of reasons:
+- Use of DAO for governance decisions & to manage risk factors.
+- Presence of multiple audits (see more).
+- Committee of elected, best-in-class validators to minimise staking risk.
+- Staking across multiple validators (i.e. current Ethereum staking mechanism only allows to choose only a single validator)
+- Use of non-custodial staking service to eliminate counterparty risk.
+
+We are constantly working on security improvements. In the [scorecard](https://scorecard.lido.fi/) you may find an  outlined set of attributes that we think are important for the decentralisation of the protocol, and how Lido is faring against these targets.  
```

Comment:
The scorecard is legit only for Lido on Ethereum. It's worth mentioning this fact.